src/orquestra/sdk/exceptions.py

```diff
@@ -204,6 +204,12 @@ class WorkflowResultsNotReadyError(NotFoundError):
     """
 
 
+class WorkflowRunIDNotFoundError(NotFoundError):
+    """Raised when we can't recover the ID for this Workflow Run."""
+
+    # Note that this is different to the WorkflowRunNotFoundError.
```

Comment:
**praise**: good comment!

Reply:
Thanks, you just reminded me I wanted to explain it further! Changed in [b233180](https://github.com/zapatacomputing/orquestra-workflow-sdk/pull/150/commits/b2331809317a9d62c4601a6434a39c16a963711c) to read:
```
# Note that this is different to the WorkflowRunNotFoundError.
# We have an ID but can't find the workflow: WorkflowRunNotFoundError
# We have a workflow but can't recover the ID: WorkflowRunIDNotFoundError
```